reference/UiTPAS.v2.json

```diff
@@ -1478,6 +1478,108 @@
         ]
       }
     },
+    "/passholders/membership-options": {
+      "get": {
+        "summary": "Retrieve UiTPAS membership options",
+        "responses": {
+          "200": {
+            "description": "OK\n\nA sorted list of possible new membership options is returned. The most relevant options (based on postal code) is on top of the list.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "price": {
+                        "type": "number"
+                      },
+                      "cardSystem": {
+                        "$ref": "#/components/schemas/CardSystem"
+                      }
+                    }
+                  }
+                },
+                "examples": {
+                  "Example": {
+                    "value": [
+                      {
+                        "price": 5,
+                        "cardSystem": {
+                          "id": 1,
+                          "name": "UiTPAS Dender"
+                        }
+                      },
+                      {
+                        "price": 5,
+                        "cardSystem": {
+                          "id": 8,
+                          "name": "UiTPAS Gent"
+                        }
+                      },
+                      {
+                        "price": 5,
+                        "cardSystem": {
+                          "id": 12,
+                          "name": "UiTPAS Oostende"
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request. Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/invalid-insz-number\n\nThe detail property might include more information for the client developer.\n",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          }
+        },
+        "operationId": "get-passholders-membership-options",
+        "description": "<!-- theme: warning -->\n\n> WARNING\n>\n> Experimental API. Breaking changes might be introduced in future updates.\n  \nRetrieve the membership options for a given insz number and email address.\n\nThe caller of this request must have `PASSHOLDER_WRITE` permission.",
```

Comment:
> Retrieve the membership options for a given insz number and email address.

Ontbreekt het email adres in de query parameters of is het niet meer relevant en mag het weg uit de description? Of wordt het op een andere manier doorgegeven?